### PR TITLE
Add AssaultCube

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/assaultcube/AC/issues/452
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/assaultcube/AC/pull/453

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.cubers.assault.AssaultCube

**Review by barthalion — APPROVED** [review]

**Comment by pizzadude** [comment]
> Multiplayer shooter
> Your Flatpak doesn't have network access, when added it doesn't even connect to the server properly

File: net.cubers.assault.AssaultCube.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+modules:
+  - name: assaultcube
+    buildsystem: simple
+    subdir: source/src
+    sources:
+      - type: git
+        url: https://github.com/assaultcube/AC.git
+        tag: v1.3.0.2
+        commit: 1ece5af7533983bb8f827616381d282470793d90
```
[the remark this answers is not in the document]
Comment (Mailaender) [reply net.cubers.assault.AssaultCube.yml:27]: Added https://github.com/flathub/net.cubers.assault.AssaultCube/commit/b2db391b1b908c5dedd0bcc36f0b91934f1ee39a

**Pull request closed without merging**